elixir/apps/web/lib/web/live/sites/edit.ex

```diff
@@ -32,65 +31,9 @@ defmodule Web.Sites.Edit do
       <:content>
         <div class="py-8 px-4 mx-auto max-w-2xl lg:py-16">
           <.form for={@form} phx-change={:change} phx-submit={:submit}>
+            <.input type="hidden" name="group[routing]" value="managed" />
```

Comment:
What do you think of making this:
```
<.input type="hidden" field={@form[:routing]} value="managed" />
```

Was just thinking it follows along with `name` field and would also prevent any issues if we ever needed to change this from `group` to something else in the future.